webapi/src/main/scala/org/knora/webapi/responders/v1/ValueUtilV1.scala

```diff
@@ -452,9 +452,15 @@ class ValueUtilV1(private val settings: SettingsImpl) {
         if (valueProps.standoff.nonEmpty) {
             // there is standoff markup
 
-            if (valueProps.standoffClassesWithDataType.isEmpty || valueProps.standoffAllPropertyEntities.isEmpty) {
-                throw NotFoundException(s"Ontology information about standoff entities is missing")
-            }
+            // get the Iri of the mapping
+            val mappingIri = valueProps.literalData.getOrElse(OntologyConstants.KnoraBase.ValueHasMapping, throw InconsistentTriplestoreDataException("no mapping Iri associated with standoff")).literals.head
+
+            // get the mapping and the realted standoff entities
+            val mappingResponseFuture: Future[GetMappingResponseV1] = (responderManager ? GetMappingRequestV1(mappingIri = mappingIri, userProfile = userProfile)).mapTo[GetMappingResponseV1]
+
+            // TODO: very very very bad
+            // FIXME: remove blocking code and return a Future instead!!!!!!!
+            val mappingResponse = Await.result(mappingResponseFuture, 2.seconds)
```

Comment:
Great. I am trying to change the behaviour for internal linking between standoff nodes today.